bin/bundle

```diff
@@ -1,3 +1,114 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
-load Gem.bin_path('bundler', 'bundle')
+# frozen_string_literal: true
```

Comment:
Regenerated the bundle binstub because bundler told me to when running the `bin/rdbg` binstub.